proxy/manager.go

```diff
@@ -24,7 +33,7 @@ func (proxy *Proxy) manage(ctx context.Context, w http.ResponseWriter, req *http
 	proxyReq.Header = req.Header
 
 	client := &http.Client{}
-	babbageResponse, err := client.Do(proxyReq)
+	clientResponse, err := client.Do(proxyReq)
```

Comment:
If anything, this is a server response. The proxy is the client in this scenario. Maybe `serviceResponse`?